__tests__/engine.test.ts

```diff
@@ -417,6 +417,103 @@ test('if or', () => {
   expect(ctx.render()).toEqual('B');
 });
 
+test('include', () => {
+  const engine = newEngine();
+  let inst: Code = [O.ROOT, 1, [
+    [O.INCLUDE, 'foo.html', 0],
+    [O.VARIABLE, [['@a']], 0],
+    [O.VARIABLE, [['b']], 0]
+  ], O.EOF];
+
+  // partial just defines a variable and outputs a string
+  const partials: Partials = {
+    'foo.html': [O.ROOT, 1, [
+      [O.BINDVAR, '@a', [['a']], 0],
```

Comment:
Would defining a variable with `EVAL` work the same way here?

Reply:
Yep, I just updated that test to include an `EVAL`-based case.